tests/drivers/sensor/i3g4250d/src/main.c

```diff
@@ -0,0 +1,71 @@
+/* On device tests for I3G4250D gyro driver
+ *
+ * Copyright (c) 2021 Jonathan Hahn
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#include <ztest.h>
+#include <device.h>
+#include <drivers/sensor.h>
+
+const struct device *dev;
+
+const struct device *get_sensor_devcie(void)
+{
+	const struct device *dev = device_get_binding(DT_LABEL(DT_INST(0, st_i3g4250d)));
+
+	zassert_true(device_is_ready(dev), "st i3g4250d not found");
+
+	return dev;
+}
+
+static void test_sensor_initialization(void)
+{
+	const struct device *dev = get_sensor_devcie();
+
+	/* check driver initialization */
+	zassert_not_null(dev, "failed: dev is null.");
+}
```

Comment:
this test can likely be more generic (all devices actually)

Reply:
@MaureenHelm @galak can you comment here?

Reply:
@gmarull I don't quite understand your question. Are you suggesting a generic gyro sensor test instead of one specific to i3g4250d?

Reply:
@MaureenHelm, @gmarull I have removed the i3g4250d specific test from this PR and added a sample instead. What do you think?